src/components/results/candidate-results/personality-dimension-details/index.js

```diff
@@ -0,0 +1,83 @@
+import PropTypes from "prop-types";
+import {Component} from "react";
+import {detailWithPerspective} from "lib/helpers";
+import {rgba} from "lib/helpers/color";
+import TraitifyPropTypes from "lib/helpers/prop-types";
+import withTraitify from "lib/with-traitify";
+import DetailsList from "./details-list";
+import style from "./style";
+
+class PersonalityDimensionDetails extends Component {
```

Comment:
I think this component was folded over into components/results/personality/dimension/details